tests/basic/pure_test.go

```diff
@@ -4132,3 +4132,111 @@ var _ = Describe("{CreateNewPoolsWhenFadaFbdaVolumeCreationInProgress}", func()
 	})
 
 })
+
+var _ = Describe("{CreateCloneOfTheFADAVolume}", func() {
+	/*
+				https://purestorage.atlassian.net/browse/PTX-24002
+			    1.Deploy a FADA app
+		        2.Take the corresponding pxctl volume of the pvc and try to clone it
+		        3.Check the corresponding volume clone is available in FA backend
+		        4.Delete the FADA app
+
+	*/
+	JustBeforeEach(func() {
+		log.Infof("Starting Torpedo tests ")
+		StartTorpedoTest("CreateCloneOfTheFADAVolume",
+			"Create Clone of the FADA Volume and verify the status",
+			nil, 0)
+	})
+	itLog := "Px Volume Resize in parallel to FADA/FBDA Volume Resize ( PVC Resize )"
```

Comment:
Update the log message to reflect the testcase

Reply:
updated